target/bin/delmailuser

```diff
@@ -67,7 +67,7 @@ do
 
     * )
       __usage
-      _exit_with_error "The option ${OPT} is unknown."
+      _exit_with_error "The option '${OPT}' is unknown."
```

Comment:
Not this line specifically, but right above it I noticed the case statement style is not what you've been adjusting to. Just leaving this as a note for you so you're aware of it due to this PR already making such a change elsewhere.

Reply:
👍🏼

Reply:
Done in 08a05debd67e69c3e5010be547c0cd0ca65a28c7